README.md

```diff
@@ -2,554 +2,59 @@
-kubectl create configmap my-test --from-file /path/to/my/test.js
-```
-
-***Note: there is a character limit of 1048576 bytes to a single configmap. If you need to have a larger test file, you'll need to use a volumeClaim or a LocalFile instead***
-
-#### VolumeClaim
-
-There is a sample avaiable in `config/samples/k6_v1alpha1_k6_with_volumeClaim.yaml` on how to configure to run a test script with a volumeClaim.
-
-If you have a PVC with the name `stress-test-volumeClaim` containing your script and any other supporting file(s), you can pass it to the test like this:
+## Contributing
 
-```yaml
-spec:
-  parallelism: 2
-  script:
-    volumeClaim:
-      name: "stress-test-volumeClaim"
-      file: "test.js"
-```
-
-***Note:*** the pods will expect to find script files in `/test/` folder. If `volumeClaim` fails, it's the first place to check: the latest initializer pod does not generate any logs and when it can't find the file, it will terminate with error. So missing file may not be that obvious and it makes sense to check it manually. See #143 for potential improvements.
-
-##### Example directory structure while using volumeClaim
-
-```
-├── test
-│   ├── requests
-│   │   ├── stress-test.js
-│   ├── test.js
-```
-
-In the above example, `test.js` imports a function from `stress-test.js` and they would look like this:
-
-```js
-// test.js
-import stressTest from "./requests/stress-test.js";
-
-export const options = {
-  vus: 50,
-  duration: '10s'
-};
-
-export default function () {
-  stressTest();
-}
-```
-
-```js
-// stress-test.js
-import { sleep, check } from 'k6';
-import http from 'k6/http';
-
-
-export default () => {
-  const res = http.get('https://test-api.k6.io');
-  check(res, {
-    'status is 200': () => res.status === 200,
-  });
-  sleep(1);
-};
-```
-
-#### LocalFile
-
-There is a sample avaiable in `config/samples/k6_v1alpha1_k6_with_localfile.yaml` on how to configure to run a test script inside the docker image.
-
-***Note: if there is any limitation on usage of volumeClaim in your cluster you can use this option, but always prefer the usage of volumeClaim.***
-
-### Executing tests
+### Requests and feedback
 
-Tests are executed by applying the custom resource `TestRun` to a cluster where the operator is running. The properties
-of a test run are few, but allow you to control some key aspects of a distributed execution.
+We are always interested in your feedback! If you encounter problems during the k6 Operator usage, check out the [troubleshooting guide](TODO). If there are questions about the k6 Operator usage, you can write them on [Grafana community forum](https://community.grafana.com/c/grafana-k6/k6-operator/73).
 
-```yaml
-# k6-resource.yml
+For new feature requests and bug reports, consider opening an issue in this repository. First check the [existing issues](https://github.com/grafana/k6-operator/issues) in case a similar report already exists. If it does, add a comment about your use case or upvote it.
```

Comment:
```suggestion
For new feature requests and bug reports, consider opening an issue in this repository. First, check the [existing issues](https://github.com/grafana/k6-operator/issues) in case a similar report already exists. If it does, add a comment about your use case or upvote it.
```